modules/ztp-site-cleanup.adoc

```diff
@@ -17,9 +17,20 @@ You can remove a managed site and the associated installation and configuration
 .Procedure
 
 . Remove a site and the associated CRs by removing the associated `SiteConfig` and `PolicyGenerator` or `PolicyGentemplate` files from the `kustomization.yaml` file.
+
+. Add the following `syncOptions` field to your `SiteConfig` application.
++
+[source,yaml]
+----
+kind: Application
+spec:
+  syncPolicy:
+    syncOptions:
+    - PrunePropagationPolicy=background
```

Comment:
See OpenShift documentation guidelines for [YAML truncating](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#yaml-formatting-for-kubernetes-and-openshift-api-objects:~:text=When%20truncating%20YAML). 
```suggestion
    - PrunePropagationPolicy=background
#...    
```